tests/conftest.py

```diff
@@ -15,4 +15,4 @@ def exhaustive_example():
 @pytest.fixture(scope="module")
 def fusion_example():
     """Create test fixture for example of fusion (additional fields excl)"""
-    return json.load(open(EXAMPLES_DIR / "fusion_example.json", "r"))
+    return json.load(open(EXAMPLES_DIR / "minimal_example.json", "r"))
```

Comment:
Wondering if we should make an issue (if it doesn't already exist) to generate the examples via fixtures from tests so that we don't have to manually update the examples